<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(67, 67, 67);</string>
  </property>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(164, 148, 255);
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(164, 148, 255);
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(164, 148, 255);
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_plus">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 216, 21);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_min">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 216, 21);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(164, 148, 255);
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_x">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 216, 21);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(164, 148, 255);
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_9">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(164, 148, 255);
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(164, 148, 255);
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_eq">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 216, 21);</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_0">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(164, 148, 255);
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_d">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 216, 21);</string>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 216, 21);</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(164, 148, 255);
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(164, 148, 255);
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(218, 255, 33);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_square">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 216, 21);</string>
   </property>
   <property name="text">
    <string>x²</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3square">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 216, 21);</string>
   </property>
   <property name="text">
    <string>x³</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sin">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 216, 21);</string>
   </property>
   <property name="text">
    <string>sin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cos">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 216, 21);</string>
   </property>
   <property name="text">
    <string>cos</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
